code/addons/a11y/package.json

```diff
@@ -61,47 +49,40 @@
   ],
   "scripts": {
     "check": "../../../scripts/prepare/check.ts",
-    "prep": "../../../scripts/prepare/bundle.ts"
+    "prep": "../../../scripts/prepare/addon-bundle.ts"
   },
   "dependencies": {
     "@storybook/addon-highlight": "workspace:*",
+    "axe-core": "^4.2.0"
+  },
+  "devDependencies": {
     "@storybook/channels": "workspace:*",
     "@storybook/client-logger": "workspace:*",
     "@storybook/components": "workspace:*",
-    "@storybook/core-events": "workspace:*",
     "@storybook/global": "^5.0.0",
     "@storybook/manager-api": "workspace:*",
     "@storybook/preview-api": "workspace:*",
     "@storybook/theming": "workspace:*",
     "@storybook/types": "workspace:*",
-    "axe-core": "^4.2.0",
-    "lodash": "^4.17.21",
-    "react-resize-detector": "^7.1.2"
-  },
-  "devDependencies": {
     "@testing-library/react": "^11.2.2",
+    "lodash": "^4.17.21",
+    "react": "^16.8.0",
+    "react-dom": "^16.8.0",
```

Comment:
Did you check if the `react-dom` dep is needed truly in these addons?

It's not a big deal at all having them as devDependencies of course, no harm to users.

Reply:
no, I added it as a todo to my cooldown period.

> - [ ]  Remove `react-dom` from `package.json`s when not needed
> - [ ]  Audit dependencies of frameworks+renderers, they might have a lot of unused `@storybook` dependencies we can remove